tests/kokkos-based/helpers.hpp

```diff
@@ -108,6 +108,17 @@ mdspan_t make_mdspan(ValueType *data, std::size_t ext0, std::size_t ext1) {
   return mdspan_t(data, ext0, ext1);
 }
 
+template<class A_t, class ValueType = typename A_t::value_type>
+void set(A_t A, ValueType value)
```

Comment:
(optional) This seems like a helpful utility.  Would you consider overloading it for `mdspan` not of rank 2?